app/vendor/views.py

```diff
@@ -208,7 +209,20 @@ def approve_order(order_id):
         abort(400)
     order.status = Status.APPROVED
     db.session.commit()
-    # TODO send emails
+
+    merchant_id = order.merchant_id
+    merchant = Merchant.query.get(merchant_id)
```

Comment:
Up to you to import either Merchant or User (either way its another model). See earlier comment on `purchases.py`